simplematrixbotlib/api.py

```diff
@@ -307,3 +307,48 @@ async def send_image_message(self, room_id, image_filepath):
             await self._send_room(room_id=room_id, content=content)
         except:
             print(f"Failed to send image file {image_filepath}")
+
+    async def send_video_message(self, room_id, video_filepath):
+        """
+        Send a video message in a Matrix room.
+
+        Parameters
+        ----------
+        room_id : str
+            The room id of the destination of the message.
+
+        video_filepath : str
+            The path to the video on your machine.
+        """
+
+        mime_type = mimetypes.guess_type(video_filepath)[0]
```

Comment:
Is `guess_type` safe to use like that?

Reply:
This worked for the image method, and appears to work for this so far.

Reply:
> The return value is a tuple (type, encoding) where type is None if the type can’t be guessed (missing or unknown suffix) or a string of the form 'type/subtype', usable for a MIME content-type header.

sounds safe enough